src/uiprotect/api.py

```
@@ -4695,17 +4708,25 @@ async def _prime_rtsps_streams(
-                        "Failed to prime RTSPS streams for camera %s",
-                        camera.id,
-                        exc_info=True,
-                    )
-                    return
+                streams: RTSPSStreams | None = None
+                for attempt in range(RTSPS_PRIME_RETRIES + 1):
+                    try:
+                        async with asyncio.timeout(RTSPS_PRIME_TIMEOUT):
+                            streams = await self.get_camera_rtsps_streams(camera.id)
+                        break
+                    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)